packages/components/src/confirm/types.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Internal dependencies
+ */
+
+// TBD
+export interface OwnProps {
+	show: boolean;
```

Comment:
Ideally we'd prefer boolean props to start with `is`/`has`/`enable`. In this case, I think a good substitute could be `isVisible` ?

Reply:
👍🏻